### PR TITLE
tracee: signatures-dir accept multiple values

### PR DESCRIPTION
**Pull request by josedonizetti**
### 1. Explain what the PR does

Fix https://github.com/aquasecurity/tracee/issues/3179

```
Author: Jose Donizetti <jdbjunior@gmail.com>
Date:   Wed Jun 14 20:48:42 2023 -0300

    feat: signatures-dir accept multiple values
```

### 2. Explain how to test it

 The tracee-rules binary should be working as before:

```
mkdir /tmp/test

./dist/tracee-rules --help # should still only support one dir
./dist/tracee-rules --list 
./dist/tracee-rules --list --rules-dir=/tmp # should find no signatures
```

The unified binary should now support multiple directories:

```
mkdir /tmp/test

./dist/tracee list
./dist/tracee list --signatures-dir=/tmp/test # should not find any signatures
./dist/tracee list --signatures-dir=/tmp/test --signatures-dir=./dist/signatures # should find normal signatures

./dist/tracee --signatures-dir=/tmp/test -f e=anti_debugging # should fail because it can't find the signatures
./dist/tracee --signatures-dir=/tmp/test --signatures-dir=./dist/signatures  -f e=anti_debugging # should work normally
```

### CODE REVIEW

File: cmd/tracee-rules/main.go [hunk 1]
```diff
@@ -54,10 +54,15 @@ func main() {
 				return fmt.Errorf("invalid target specified: %s", strings.ToLower(c.String("rego-runtime-target")))
 			}
 
+			var rulesDir []string
+			if c.String("rules-dir") != "" {
+				rulesDir = []string{c.String("rules-dir")}
```
Comment (yanivagman) [inline cmd/tracee-rules/main.go:59]: Shouldn't this be GetStringArray?

Comment (josedonizetti) [reply cmd/tracee-rules/main.go:59]: We are not changing `tracee-rules`, only `tracee`.

**Review by yanivagman — APPROVED** [review]
LGTM

**Review by rafaeldtinoco — APPROVED** [review]
LGTM

**Pull request merged**